### PR TITLE
Content-wip-housing-parity

### PR DESCRIPTION
**Pull request by peggygannon**
# DO NOT MERGE: REQUIRES SME APPROVAL

## Page to edit
Home loans/Housing section

## Origin of request (internal/stakeholder/user feedback)
Home Loans parity edits

## Description of what's needed (edits/link changes/additions)
Assess and implement changes to bring parity between new VA.gov pages and legacy VA.gov pages in the Home Loans section.

### CODE REVIEW

**Comment by peggygannon** [comment]
@bethpotts The housing parity edits are ready for your review.

File: pages/housing-assistance/index.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ social:
         links:
         - url: https://iris.custhelp.va.gov/app/ask
           label: "Ask a question online"
+        - url: https://www.hud.gov/i_want_to/talk_to_a_housing_counselor 
+          label: "Talk to a housing counselor"
       - subhead: Call us
```
Comment (ncksllvn) [inline pages/housing-assistance/index.md]: @bethpotts I had to resolve a conflict with `master` around here. Can you make sure it looks okay?

Comment (bethpotts) [reply pages/housing-assistance/index.md]: @ncksllvn I'm asking Mikki. Sorry, just seeing this...

**Comment by ncksllvn** [comment]
Sorry about the issues you all were seeing here with build! They seem to be resolved now but LMK otherwise.

File: pages/housing-assistance/index.md (hunk 1 above)
Comment (bethpotts) [reply pages/housing-assistance/index.md]: @peggygannon is going to remove this "Talk to a housing counselor" link.

**Comment by timwright12** [comment]
closing due to age, conflicts, and inactivity

**Pull request closed without merging**